src/admin-views/base_admin_panel.php

```diff
@@ -1,40 +1,42 @@
+<?php
+$total_tickets = Tribe__Tickets__Tickets_Handler::instance()->get_total_event_capacity( $post_id );
+$container_class = 'tribe_sectionheader ticket_list_container';
+$container_class .= ( empty( $total_tickets ) ) ? ' tribe_no_capacity' : '' ;
+?>
 <div id="tribe_panel_base" class="ticket_panel panel_base" aria-hidden="false">
-	<div class="tribe_sectionheader ticket_list_container">
-		<div class="ticket_table_intro">
-			<?php
-			/**
-			 * Allows for the insertion of total capacity element into the main ticket admin panel "header"
-			 *
-			 * @since TBD
-			 *
-			 * @param Post ID
-			 */
-			do_action( 'tribe_events_tickets_capacity', $post_id );
-
-			/**
-			 * Allows for the insertion of additional elements (buttons/links) into the main ticket admin panel "header"
-			 *
-			 * @since TBD
-			 *
-			 * @param int $post_id the id of the post
-			 */
-			do_action( 'tribe_events_tickets_post_capacity', $post_id );
+	<div class="<?php echo esc_attr( $container_class ); ?>">
+		<?php
+		// only show if there are tickets
+		if ( ! empty( $total_tickets ) ) :
+			?>
+			<div class="ticket_table_intro">
+				<?php
+				/**
+				 * Allows for the insertion of total capacity element into the main ticket admin panel "header"
+				 *
+				 * @since TBD
+				 *
+				 * @param int $post_id the id of the post
+				 */
+				do_action( 'tribe_events_tickets_capacity', $post_id );
 
-			$total_tickets = Tribe__Tickets__Tickets_Handler::instance()->get_total_event_capacity( $post_id );
-			// only show if there are tickets
-			if ( ! empty( $total_tickets ) ) {
+				/**
+				 * Allows for the insertion of additional elements (buttons/links) into the main ticket admin panel "header"
+				 *
+				 * @since TBD
+				 *
+				 * @param int $post_id the id of the post
+				 */
+				do_action( 'tribe_events_tickets_post_capacity', $post_id );
 				?>
 				<a id="ticket_form_view_attendees" class="ticket_form_view_attendees" href="<?php echo esc_url( $attendees_url ); ?>"><?php esc_html_e( 'View Attendees', 'event-tickets' ); ?></a>
-				<?php
-			}
-			?>
-		</div>
-
+			</div>
+		<?php endif; ?>
 		<?php
 		/**
 		 * Allows for the insertion of additional content into the main ticket admin panel before the tickets listing
 		 *
-		 * @param int Post ID
+		 * @param int $post_id the id of the post
 		 * @since TBD
```

Comment:
for this hook and the one after please move @since up it is existing, but a quick fix.